Add mixology-helper plugin

BlueSoapTurtle opened the pull request:
Added version 1.0 of the Mixology Helper plugin. 
This plugin helps players optimize their performance in the Mastering Mixology minigame by highlighting key game elements, picking the best potion order based on Herblore level and configured priorities(exp or specific resin focused), and tracking progress through customizable infoboxes. 
It also provides notifications for special events like mature digweed spawns.

DapperMickie commented:
This is mixology plugin number 5 or 6, we're starting to collaborate over at https://github.com/hex-agon/mastering-mixology. 
Feel free to join our discussion in the discord, there's a thread created for the Mixology plugin creators to collaborate.

BlueSoapTurtle commented:
Pushed a new change that adds a few more changes and improvements:
- Fixed order list not refreshing on login or when you level up
- Fixed some display issue
- Added tooltips to herbs and unfinished pots in the bank to show how much paste they give in total, each and the gp/paste
- Added banked paste to the panel

I also udpated the readme with some more photos and info if that helps with the review:
https://github.com/BlueSoapTurtle/mixology-helper

raiyni commented:
To reiterate what DapperMickie said, we already have  one has already been merged. But you are welcome to contribute to it or join the discussion in discord.

BlueSoapTurtle commented:
@raiyni I read this on the contribution guidelines
_We recommend contributing to existing plugins if the author(s) are accepting contributions_

So assumed i could still make and maintain my own plugin in case I wanted to, if that's not the case then I apologize, I see the other plugin mentioned doesn't have the same functionality as this one does so I don't really follow, but I am new to runescape and runelite so I don't want to do something wrong ofcourse.
